data/ecosystem/vendors.yaml

```diff
@@ -457,3 +457,9 @@
   contact: contact@rizhiyi.cn
   oss: false
   commercial: true
+- name: GuanChaYi by LogEase
+  nativeOTLP: true
+  url: https://logease.cn/en-logease-observability-platform/
+  contact: contact@yottabyte.cn
+  oss: false
+  commercial: true
```

Comment:
how is this different from the entry for rizhiyi? the pages use the same language and if I switch languages I can go back and forth between LogEase and rizhiyi

Reply:
Already updated, would you please help to review it, thanks @svrnm